### ISSUE TITLE
PyPi search command

### ISSUE THREAD

**Issue by pedrofracassi** [SwitchbladeBot/switchblade#653]
https://pypi.org/

**Comment by pedrofracassi** [comment]
Working on it.

**Comment by pedrofracassi** [comment]
https://warehouse.pypa.io/api-reference/json/

**Comment by pedrofracassi** [comment]
I'll be using cheerio to scrape PyPi's search page, as the API does not have a search endpoint.

**Comment by pedrofracassi** [comment]
https://stackoverflow.com/questions/32655076/cheerio-jquery-selectors-how-to-get-a-list-of-elements-in-nested-divs

**Comment by pedrofracassi** [comment]
https://api.jquery.com/map/

**Comment by SamC302** [comment]
@pedrofracassi Can I work on this?

**Comment by pedrofracassi** [comment]
Sure!

**Comment by pedrofracassi** [comment]
Hey, do you still plan to work on this?

**Comment by SamC302** [comment]
Yes, sorry about the delay. I've been working on understanding how to make a command in switchblade. I think I've got it now after looking at a few examples.  I think I can get it done within the next few days. Is that ok?

**Comment by pedrofracassi** [comment]
Totally ok! Glad to know you're still interested :)
